### ISSUE TITLE
Improve Accessibility for Loading Button

### ISSUE THREAD

**Issue by chimobi-justice** [snaps-ui/snaps-ui#17]
### Description

The loading button should announce state changes to screen readers. For example, when isLoading is true, we should provide aria-busy and optionally aria-live attributes.

### Steps to reproduce

2. Click on '...'
3. Scroll down to '...'

### Snaps UI Version

0.0.0.-development 

### Operating System

- [x] macOS
- [x] Windows
- [x] Linux
